frontend/src/tests/import-export-data/diff-data.js

```diff
@@ -0,0 +1,603 @@
+                end_date: "2020-12-31T00:00:00.000Z",
+                note: null,
+                contract_override_pdf: null,
+                active_offer_status: null,
+                active_offer_url_token: null,
+                active_offer_recent_activity_date: null,
+                active_offer_nag_count: null,
+                hours: 70,
+                position: {
+                    id: 1388,
+                    position_code: "MAT136H1F",
+                    position_title: "Calculus II",
+                    start_date: "2020-02-10T00:00:00.000Z",
+                    end_date: "2020-12-31T00:00:00.000Z",
+                    hours_per_assignment: 70,
+                    qualifications: null,
+                    duties: null,
+                    ad_hours_per_assignment: null,
+                    ad_num_assignments: null,
+                    ad_open_date: null,
+                    ad_close_date: null,
+                    desired_num_assignments: null,
+                    current_enrollment: null,
+                    current_waitlisted: null,
+                    instructors: [
+                        {
+                            id: 419,
+                            first_name: "Emily",
+                            last_name: "Garcia",
+                            email: "emily.garcia@utoronto.ca",
+                            utorid: "garciae",
+                        },
+                        {
+                            id: 418,
+                            first_name: "Henry",
+                            last_name: "Smith",
+                            email: "hery.smith@utoronto.ca",
+                            utorid: "smithh",
+                        },
+                    ],
+                    contract_template: {
+                        id: 889,
+                        template_file: "default-template.html",
+                        template_name: "Regular",
+                    },
+                    instructor_preferences: [],
+                },
+                applicant: {
+                    id: 1763,
+                    first_name: "Harry",
+                    last_name: "Potter",
+                    email: "harry@potter.com",
+                    phone: null,
+                    utorid: "potterh",
+                    student_number: "999666999",
+                },
+                wage_chunks: [
+                    {
+                        id: 1973,
+                        assignment_id: 1637,
+                        start_date: "2020-02-10T00:00:00.000Z",
+                        end_date: "2020-12-31T00:00:00.000Z",
+                        hours: 70,
+                        rate: 50,
+                    },
+                ],
+            },
+        },
+    ],
+};
+
+// modified instructor data used for computing the diff with initial objects
+export const modifiedInstructorData = reduxStoreData.instructors.map(
```

Comment:
This is good! Move this creation part into the actual test function, and I think this PR will be ready :-)

BTW, what's the difference between the `reduxStoreData` and what is stored in all the `.json` files spread around?

Reply:
For `json`s used in export tests, for example, `assignments` are designed to reflect any possible situations how output file should look like (depending on `assignment.hours`, `assignment.wage_chunks`). For `json` files used in import tests which are spread around, they may have missing fields, redundant fields, or strange chars. All the `json`s are designed to cover some edge cases.